google-login-plugin/src/com/google/gct/login/stats/UsageTrackerManager.java

```diff
@@ -0,0 +1,50 @@
+package com.google.gct.login.stats;
+
+import com.intellij.ide.util.PropertiesComponent;
+import com.intellij.openapi.diagnostic.Logger;
+
+/**
+ * Stores the users choice to opt in/out of sending usage metrics via the Google Usage Tracker.
+ */
+public class UsageTrackerManager {
```

Comment:
could this be a service? I'd like us to try and avoid using static methods for everything.